### PR TITLE
FUSETOOLS-1837 - fuse nature automatically added on importing a fuse …

### PR DESCRIPTION
**Pull request by lhein**
…quickstart

### CODE REVIEW

**Comment by lhein** [comment]
testPR

File: editor/plugins/org.fusesource.ide.project/src/org/fusesource/ide/project/camel/maven/CamelProjectConfigurator.java [hunk 1]
```diff
@@ -205,4 +230,94 @@ private String inferCamelVersionFromDependencies(MavenProject mavenProject) {
...
+	 * adds the fuse camel and java nature to the project if needed
+	 * 
+	 * @param project
+	 */
+	private void addFuseNature(IProject project, IProgressMonitor monitor) throws CoreException {
+		boolean camelNatureFound = false;
+		boolean javaNatureFound = false;
+		IProjectDescription projectDescription = project.getDescription();
+		String[] ids = projectDescription.getNatureIds();
+		for (String id : ids) {
```
Comment (apupier) [inline editor/plugins/org.fusesource.ide.project/src/org/fusesource/ide/project/camel/maven/CamelProjectConfigurator.java]: we might replace with Java 8 stream:
javaNatureFound = Arrays.stream(ids).anyMatch(s -> JavaCore.NATURE_ID.equals(s));
camelNatureFound = Arrays.stream(ids).anyMatch(s -> RiderProjectNature.NATURE_ID.equals(s));

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
Aurelien, added your suggestion. Feel free to revisit the PR and merge if all is fine and Jenkins says OK

**Pull request merged**